README.md

```diff
@@ -153,6 +153,31 @@ activate.sh
 environment.ps1
 ```
 
+#### Custom pixi-pack executable path
+
+When creating a self-extracting binary, you can specify a custom path or URL to a `pixi-pack` executable to avoid downloading it from the default location:
+https://github.com/Quantco/pixi-pack/releases/download/.
```

Comment:
```suggestion
When creating a self-extracting binary, you can specify a custom path or URL to a `pixi-pack` executable to avoid downloading it from the [default location](https://github.com/Quantco/pixi-pack/releases/download).
```